response_operations_ui/controllers/sample_controllers.py

```diff
@@ -51,3 +52,22 @@ def upload_sample(short_name, period, file):
                  sample_id=sample_summary['id'])
 
     return sample_summary
+
+
+def search_samples_by_postcode(postcode) -> dict:
+    url = f'{app.config["SAMPLE_URL"]}/samples/sampleunits'
+    response = requests.get(url=url,
+                            auth=app.config['SAMPLE_AUTH'],
+                            params={'postcode': postcode})
+
+    logger.debug("Searching for samples by postcode", postcode=postcode)
+    try:
+        response.raise_for_status()
+    except HTTPError:
+        if response.status_code == 404:
+            logger.error("No samples were found for postcode", postcode=postcode)
```

Comment:
Doesn't seem like we should log at error here. It's a pretty normal scenario that someone puts in a bad postcode or one that has no samples

Reply:
As in case_controller.py: debug/info.